docs/types/txikijs.d.ts

```diff
@@ -579,17 +572,10 @@ declare global {
         */
         function open(path: string, flags: string, mode?: number): Promise<FileHandle>;
         
-        /**
-        * Removes the directory at the given path.
-        *
-        * @param path Directory path.
-        */
-        function rmdir(path: string): Promise<void>;
-        
         interface MkdirOptions {
```

Comment:
Probably it should be named MakeDirOptions

Reply:
👍